docs/rmis.md

```diff
@@ -0,0 +1,243 @@
+
+### Types of Hosting
+
+-   Hosting on the PEA / consultant's own server
+
+-   Cloud-based hosting of the R / MIS (Software-as-a-Service, SAAS)
+
+### Software cost structure model:
+
+ Is an initial, one-time payment with no / low ongoing (license) costs
+ preferred? Can ongoing (license) costs be covered beyond the project
+ period? Are customizations possible?
+
+ It must be decided which software category is appropriate for the
+ setup of the R / MIS. In all three categories, open- source and
+ proprietary solutions are available. In general, **open-source solutions are preferable** ([> RMMV Guidebook Section 2.2.2](https://www.kfw-entwicklungsbank.de/Service/Publications-Videos/Publications-by-topic/Digitalisation/RMMV-Guidebook/)).
+
+#### Customized software
+
+ Tailor-made solutions meeting the specific needs and requirements of a
+ project, portfolio, or organization.
+
+*Advantages:*
+
+ - Customized to reflect the exact needs, structures, and features
+ - Suitable for complex project setups with a high number of users and
+ access / approval levels
+ - Connectors for exchange with other tools can be programmed
+ -  Usually lower or no licensing costs (none if open source) -- Usually higher costs for (initial) setup (compared to off-the-shelf tools)
+
+*Disadvantages:*
+
+-   Setup requires IT expert resources
+-   Usually longer implementing period
+-   Ongoing costs for support and maintenance
+
+ *Potentially suitable for Case B (if existing R / MIS is already
+ customized or self-developed), Case C (if complex project, many
+ users), Case A (if complex procedures, many users, AND no good
+ sector-specific, off-the-shelf solutions exist).*
+
+#### Modifiable off-the-shelf software (MOTS)
+
+ Standard tools that can be modified on software source code level.
+
+*Advantages:*
+
+- Fast setup as MOTS are based on available software solutions with
+ existing modules
+- Customizations possible by the purchaser or by another party (if
+ open source)
+
+
+*Disadvantages:*
+
+-  Customizations possible only by the vendor (if proprietary)
+-   Setup and customizations require IT expert resources
+
+-   Ongoing costs for support and maintenance
+
+-   License costs based on time, user numbers, and support (if
+    proprietary)
+
+ *Potentially suitable for all cases if the project's / PEA's proce-
+ dures being digitalized / established are not too complex and do not
+ require too much customization.*
+
+#### Commercial off-the-shelf software (COTS)
+
+ Standard tools that can be used "as is" without the option to be
+ modified on software source code level.
+
+ \+ Available "out of the box" with basic functionality
+
+ \+ Suitable for simple project setups that do not require
+ customization
+
+## Interoperability Requirements
+
+ In which formats and structures should data be stored in the R / MIS to allow for an exchange with others relevant to the pro- ject / organization? In general, the R / MIS must fit into the already
```

Comment:
the text in this line should be merged with the next few lines so they have the same length. Not sure how it is displayed on the website. But I think we can make it look more coherent here?